nifi-commons/nifi-site-to-site-client/src/test/groovy/org/apache/nifi/remote/client/PeerSelectorTest.groovy

```diff
@@ -71,6 +72,11 @@ class PeerSelectorTest extends GroovyTestCase {
 
     }
 
+    private static String buildRemoteInstanceUris(List<String> nodes = DEFAULT_NODES) {
+        String remoteInstanceUris = new String("http://").concat(nodes.join(":8443/nifi-api,http://")).concat(":8443/nifi-api");
```

Comment:
Should just use `String remoteInstanceUris = "http://" + nodes.join(":8443/nifi-api,http://" + ":8443/nifi-api");`... no need to use new String() or concat methods.

Reply:
changed to ```String remoteInstanceUris = "http://" + nodes.join(":8443/nifi-api,http://") + ":8443/nifi-api";```